irods_audit_elk_stack/README.md

```diff
@@ -0,0 +1,106 @@
+# iRODS Audit Demonstration ELK Stack Container
+
+This is an ELK-like stack container used for demonstrating the
+[audit rule engine plugin](https://github.com/irods/irods_rule_engine_plugin_audit_amqp>) for
```

Comment:
remove trailing `>`

Reply:
Nice catch! I was originally writing this in reStructuredText but switched to Markdown when I realized RST doesn't let you specify column alignment in tables, and this is left-over from the RST version.